lib/git_server/models/concept.rb

```diff
@@ -0,0 +1,65 @@
+module GitServer
+  class Concept
+    extend Mandate::Memoize
+
+    # TODO: repo_url can be removed once we're out of a monorepo
+    def initialize(track_slug, concept_slug, git_sha = "HEAD", repo_url: nil)
+      @repo = Repository.new(track_slug, repo_url: repo_url)
+      @track_slug = track_slug
+      @concept_slug = concept_slug
+      @git_sha = git_sha
+    end
+
+    memoize
+    def data
+      {
+        about: about,
+        links: links
+      }
```

Comment:
I reckon move this into `RetrieveConceptData` and expose `about` and `links` and public methods on Concept.

But equally, this is totally fine so let's maybe keep it as a future tweak once we actually standardise everything!

Reply:
Agreed. I've just copy-pasted things.